locale/en/LC_MESSAGES/django.po

```diff
@@ -4056,11 +4044,11 @@ msgid "Informatiebrief ouders"
 msgstr "Information letter for the parents"
 
 #: reviews/templatetags/documents_list.py:242
-msgid "Toestemmingsdocument observatiestudie"
-msgstr "Consent document for observational study"
+msgid "Toestemmingsdocument observatie"
+msgstr "Consent document for observation"
```

Comment:
Ik zou zeggen 'observatie-onderzoek' en 'observation research'

Reply:
Of je laat studie staan, in deze context is het volgensmij nog wel gepast. Iets verderop staat ook 'interventiestudie' bv.